src/pages/SearchHelpPage.vue

```diff
@@ -0,0 +1,195 @@
+<template>
+<div class="page grid-container full">
+  <header-section showNavSearch="true"></header-section>
+  <div class="page_body">
+    <h1>CC Search Syntax Guide</h1>
+    <p>
+      When you search, you can enter special symbols or words to your search term
+      to make your search results more precise.
+    </p>
+
+    <h2>Search for an exact match</h2>
+    <p>
+      Put a word or phrase inside quotes. For example,
+      <a href='https://search.creativecommons.org/search?q="Claude%20Monet"'>
+        <em>"Claude Monet"</em>
+      </a>.
+    </p>
+
+    <h2>Search by title</h2>
+    <p>
+      Put <em>title:</em> in front of your search term to see only results whose
+      title contains the specified term. For example,
+      <a href='https://search.creativecommons.org/search?q=title%3A%20"cats"'>
+        <em>title:"cats"</em>
+      </a>.
+    </p>
+
+    <h2>Search by license</h2>
+    <p>
+      Put <em>license:</em> in front of your search term to see only results
+      from the specified source. For example,
+      <a href='https://search.creativecommons.org/search?q=license%3A%20"by"'>
+        <em>license: "by"</em>
```

Comment:
Does this search work? It does not return any results when I try it although all the other keywords work fine. Could you double check the syntax and update or remove if needed?

Reply:
Please remove space between keyword and query.

Reply:
The space doesn't affect the query. I think it makes it clearer, with the extra separation.